mesheryctl/internal/cli/root/system/start.go

```diff
@@ -71,6 +71,9 @@ mesheryctl system start --reset
 
 // Specify Platform to deploy Meshery to.
 mesheryctl system start -p docker
+
+// Specify Provider to use.
+mesheryctl system start --provider Meshery
```

Comment:
Does the logic in this PR support multi-word providers like "CNCF Lab"? @hexxdump @alphaX86 can you confirm?